Resources/config/services.xml

```diff
@@ -152,10 +153,16 @@
             <tag name="kernel.event_listener" event="lexik_translation.event.get_database_resources" method="onGetDatabaseResources"  />
         </service>
 
+
         <!-- Command -->
-        <service id="Lexik\Bundle\TranslationBundle\Command\ImportTranslationsCommand" class="%lexik_translation.command.import_translations.class%" autowire="true">
-            <argument type="service" id="lexik_translation.translator" />
+        <service id="Lexik\Bundle\TranslationBundle\Command\ImportTranslationsCommand" class="%lexik_translation.command.import_translations.class%">
+            <argument type="service" id="translator" />
```

Comment:
I'm not sure if it's a good idea to change with the alias name, why do you want to change this?

Reply:
In previous version this command call Translation from alias 'translator'.
https://github.com/lexik/LexikTranslationBundle/pull/305/files#diff-d799211cef1edefe3a7a6cb2456ebdd3L117
We pass 'lexik_translation.translator' because 'translator' was 'private'.